data/device-tests/amd-pmc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "AMD PMC",
+  "interactive": false,
+  "steps": [
+    {
+      "emulation-url": "https://github.com/fwupd/fwupd-emulation/raw/refs/heads/main/amd-pmc/3ac3159e-6eef-5f6b-bc8d-67686b238747/setup.json",
```

Comment:
so; question -- why don't we just ship in them in the fwupd repo? i.e. in the same place as `data/device-tests/amd-pmc-emulation.json` -- then if we bisect fwupd and run the tests we've got the "right" data for the bisect to pass.

Reply:
Good point!  I've reworked it.